src/main/cpp/mechanisms/noteManager/decoratormods/launcherToPlacerBackState.cpp

```diff
@@ -14,60 +14,60 @@
 // This file was automatically generated by the Team 302 code generator version 1.2.1.0
-// Generated on Monday, January 29, 2024 6:39:38 AM
+// Generated on Friday, February 2, 2024 7:00:50 PM
 
 // C++ Includes
 #include <string>
 
 // FRC includes
 
 // Team 302 includes
-#include "mechanisms/noteManager/generated/noteManager_placeTrap_StateGen.h"
-#include "mechanisms/noteManager/decoratormods/noteManager_placeTrap_State.h"
+#include "mechanisms/noteManager/decoratormods/launcherToPlacerBackState.h"
 #include "teleopcontrol/TeleopControl.h"
 #include "teleopcontrol/TeleopControlFunctions.h"
 #include "utils/logging/Logger.h"
 
 // Third Party Includes
 
 using namespace std;
+using namespace noteManagerStates;
 
 /// @class ExampleForwardState
 /// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
-noteManagerplaceTrapState::noteManagerplaceTrapState ( std::string stateName,
+launcherToPlacerBackState::launcherToPlacerBackState ( std::string stateName,
         int stateId,
-        noteManagerplaceTrapStateGen *generatedState,
+        noteManagerAllStatesStateGen *generatedState,
         noteManager *mech ) : State ( stateName, stateId ), m_genState ( generatedState ), m_mechanism ( mech )
 {
 }
 
-void noteManagerplaceTrapState::Init()
+void launcherToPlacerBackState::Init()
 {
-	Logger::GetLogger()->LogData ( LOGGER_LEVEL::PRINT, string ( "ArrivedAt" ), string ( "noteManagerplaceTrapState" ), string ( "init" ) );
+	Logger::GetLogger()->LogData ( LOGGER_LEVEL::PRINT, string ( "ArrivedAt" ), string ( "launcherToPlacerBackState" ), string ( "init" ) );
 
 	m_genState->Init();
 }
 
-void noteManagerplaceTrapState::Run()
+void launcherToPlacerBackState::Run()
 {
-	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("noteManagerplaceTrapState"), string("run"));
+	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("launcherToPlacerBackState"), string("run"));
 	m_genState->Run();
 }
 
-void noteManagerplaceTrapState::Exit()
+void launcherToPlacerBackState::Exit()
 {
 	m_genState->Exit();
 }
 
-bool noteManagerplaceTrapState::AtTarget()
+bool launcherToPlacerBackState::AtTarget()
 {
 	auto attarget = m_genState->AtTarget();
 	return attarget;
 }
 
-bool noteManagerplaceTrapState::IsTransitionCondition ( bool considerGamepadTransitions )
+bool launcherToPlacerBackState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	
-	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::PLACE ) && AtTarget() );
+
+	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::EXAMPLE_MECH_FORWARD ) );
```

Comment:
Why is this using an ExampleMech teleop function?

Reply:
closes #212  

That is the bases generated code as an example, not mapped to anything